README.md

```diff
@@ -125,9 +143,19 @@ action. You can also implement other personal action.
     needed for execute the brick
   - show --format direct-next: display the list of bricks that needs the output
     of the specified bricks
-  - show --format linked-previous: display the list of bricks that needs to be 
+  - show --format linked-previous: display the list of bricks that needs to be
     layed to lay your brick (as their outputs are needed directly or not)
-  - show --format linked-next: display the list of bricks that can be impacted 
+  - show --format linked-next: display the list of bricks that can be impacted
     recursively by a change in your brick
-  - show --children: display elementary bricks 
+  - show --children: display elementary bricks
   - ... others exeiac command in general
+
+## Development
+
```

Comment:
Perhaps we can add this to docs/development.md.
So we will feel free to be more verbose

Reply:
Good idea! It just didn't want to remove it all-together from the main `README.md`, but if it's fine for you I'll do that sure.